controllers/machinehealthcheck_targets.go

```diff
@@ -26,29 +26,36 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/labels"
-	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/client-go/tools/record"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	"sigs.k8s.io/cluster-api/util/conditions"
+	"sigs.k8s.io/cluster-api/util/patch"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	ownerControllerKind   = "MachineSet"
-	nodeControlPlaneLabel = "node-role.kubernetes.io/master"
+	// Condition reasons
+	hasFailureReason       = "MachineHasFailureReason"
```

Comment:
FYI, in other PRs I'm keeping Reasons in the API, using a const statement to group a condition and its own related reason